### PR TITLE
Add deprecation message for deploy tools

### PR DESCRIPTION
**Pull request by RunningLeon**
## Motivation

Since [MMDeploy](https://github.com/open-mmlab/mmdeploy) has been released, the deployment tools scattered in each OpenMMLab codebase will be deprecated in future.

## Modification

Add deprecation message in deployment tools.
Used package [colorama](https://github.com/tartley/colorama#colored-output) to print cross-platform colored text.

## BC-breaking (Optional)

None

## Use cases (Optional)

And the message would look like it in Ubuntu18.04 terminal:
image

### CODE REVIEW

**Comment by RunningLeon** [comment]
@lvhan028 Hi, any comment on this PR?

**Review by lvhan028 — APPROVED** [review]
LGTM

File: tools/deployment/onnx2tensorrt.py [hunk 1]
```diff
@@ -140,3 +140,13 @@ def parse_args():
         fp16_mode=args.fp16,
         verify=args.verify,
         workspace_size=args.workspace_size)
+
+    import warnings
+    from colorama import Fore, Back, Style
```
Comment (mzr1996) [inline tools/deployment/onnx2tensorrt.py:145]: The `colorama` is not a dependency of mmcls. Please use `try` to handle `ImportError`.
```suggestion
    try:
        from colorama import Fore, Style, Back
    except ImportError:
        Style = mmcv.ConfigDict({'BRIGHT': '\x1b[1m', 'RESET_ALL': '\x1b[0m'})
        Fore = mmcv.ConfigDict({'RED': '\x1b[31m', 'BLUE': '\x1b[34m'})
        Back = mmcv.ConfigDict({'LIGHTWHITE_EX': '\x1b[107m'})
```

**Review by mzr1996 — APPROVED** [review]
LGTM

**Pull request merged**